sky/utils/controller_utils.py

```diff
@@ -299,18 +322,22 @@ def get_controller_resources(
         specified, the controller will be launched on one of the clouds
         of the task resources for better connectivity.
     """
-    controller = Controllers.from_type(controller_type)
-    assert controller is not None, controller_type
     controller_resources_config_copied: Dict[str, Any] = copy.copy(
         controller.value.default_resources_config)
     if skypilot_config.loaded():
         # Override the controller resources with the ones specified in the
         # config.
         custom_controller_resources_config = skypilot_config.get_nested(
-            (controller_type, 'controller', 'resources'), None)
+            (controller.value.controller_type, 'controller', 'resources'), None)
```

Comment:
seems like the `controller_type` is `jobs`, while the config schema says `managed_jobs`. align those two?